README.md

```diff
@@ -14,11 +15,16 @@ gem 'bambora-client'
 
 And then execute:
 
-    $ bundle
+```bash
+bundle
+```
 
 Or install it yourself as:
 
-    $ gem install bambora-client
+```bash
+gem install bambora-client
+
```

Comment:
Is the extra space needed? Not a blocker for this PR. This could be fixed when some Markdown linting is included 👀.